central/src/App.tsx

```diff
@@ -99,7 +99,7 @@ function App() {
       domain: game.domain,
       cluster: game.cluster,
       standard: game.standard,
-      questions: questionIDSet,
+      questions: questions,
```

Comment:
the `updateGame` mutation requires question objects instead of the `questionIDSet` value that we're using in other places in `central`